src/transformStateWithClones.js

```diff
@@ -7,7 +7,47 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const TYPE_ADD_PROPERTIES = 'addProperties';
+  const TYPE_REMOVE_PROPERTIES = 'removeProperties';
+  const TYPE_CLEAR = 'clear';
```

Comment:
👍  you are using constants, that is great

But these constants should be declared outside of the function, otherwise you will re-create them on each function call